build/mpi-proxy-split/Makefile_config

```diff
@@ -0,0 +1,40 @@
+# FIXME: We should eventually dynamically choose this address in setLhMemRange.
+#        Then, we won't need this configure variable.
+MANA_USE_LH_FIXED_ADDRESS = @MANA_USE_LH_FIXED_ADDRESS@
```

Comment:
@tarunsmalviya , FYI: please see https://github.com/mpickpt/mana/issues/133 , where you'll see that they should soon have this fixed, so that you don't need a hack like MANA_USE_LH_FIXED_ADDRESS.